src/classes/ACCT_IndividualAccounts_TDTM.cls

```diff
@@ -41,6 +41,10 @@ public class ACCT_IndividualAccounts_TDTM extends TDTM_Runnable {
     */ 
     private static Hierarchy_Settings__c ContactsSettings;
     
+    private static ID orgRecTypeID = Schema.Sobjecttype.Account.getRecordTypeInfosByName().get('Business Organization').getRecordTypeId();
```

Comment:
These recordtypes aren't necessarily hardcoded, are they? What happens if the user's recordtypes are something other than the default?